### PR TITLE
US-56177 Moving new mounted files to locations

### PR DESCRIPTION
**Pull request by saikarthik528**
US-56177 Moving new mounted files to locations

Added support for

catalina.properties
prbootstrap.properties
java.security.overwrite
tomcat-web.xml

### CODE REVIEW

File: scripts/docker-entrypoint.sh [hunk 1]
```diff
@@ -289,6 +293,46 @@ else
...
+else
+  echo "No catalina.properties was specified in ${catalina_properties}. Using defaults."
+fi
+
+#
+# Copying mounted prbootstrap properties file to webapps/prweb/WEB-INF/classes
+#
+if [ -e "$prbootstrap_properties" ]; then
+  echo "Loading prbootstrap.properties from ${prbootstrap_properties}...";
+  cp "$prbootstrap_properties" ${PEGA_DEPLOYMENT_DIR}/WEB-INF/classes/
```
Comment (kishorv10) [inline scripts/docker-entrypoint.sh:311]: Just curious the other cp commands has second argument in double quotes. Should this be also changed? Are we able to test it?

Comment (saikarthik528) [reply scripts/docker-entrypoint.sh:311]: Took reference from prlog4j2 and prconfig, where files are getting copied into WEB-INF/classes, where quotes are not mentioned.

Unittests were added.

Comment (kishorv10) [reply scripts/docker-entrypoint.sh:311]: For [web.xml](https://github.com/pegasystems/docker-pega-web-ready/blob/master/scripts/docker-entrypoint.sh#L287) it is added that's where I checked. If its tested we are good.

**Review by kishorv10 — APPROVED** [review]

File: scripts/docker-entrypoint.sh (hunk 1 above)
Comment (saikarthik528) [reply scripts/docker-entrypoint.sh:311]: Tested in local as well.  It is working as expected.

**Pull request merged**